ckeditor/ckeditor.d.ts

```diff
@@ -544,8 +544,13 @@ declare namespace CKEDITOR {
     }
 
 
-    interface focusManager {
-
+    class focusManager {
+        focus(element: dom.element): void;
```

Comment:
per [docs](http://docs.ckeditor.com/#!/api/CKEDITOR.focusManager) `element` is named `currentActive` and is optional